src/model/RecurrentLSTMNetwork.lua

```diff
@@ -12,9 +12,9 @@ return function(opt, params)
    local batchNormalization = opt.batchNormalization or false
    local maxBatchNormalizationLayers = opt.maxBatchNormalizationLayers or 10
 
-   -- container:
+   -- containers:
    params = params or {}
-   layers = layers or {}
```

Comment:
This line was something I added in the original PR since I didn't really understand how the other models used layers yet. Since the LSTM is implemented in autograd it can't be concatenated with a sequence of nn modules so passing in preceding layers doesn't make sense. While the batch norm layers here share parameters, they are all disconnected and applied independently at each time step.